scripts/build/ci-build/Dockerfile

```diff
@@ -126,7 +126,7 @@ RUN apt-get update && \
         gcc             \
         g++             \
         git             \
-	jq		\
+	      jq		          \
```

Comment:
what is this?

Reply:
Heh, it's me aligning this, but missing that the first whitespace on the `jq` was a tab and not four spaces